public/includes/find-house.php

```diff
@@ -0,0 +1,65 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sqlQuery = "SELECT * FROM `states` WHERE 1";
+    $result = mysqli_query($mysqli,$sqlQuery);
+    echo "<option selected disabled>State</option>";
+    while ($row = mysqli_fetch_array($result, MYSQLI_ASSOC)) {
+        echo "<option value='".$row['state_id']."'>".$row['state_name']."</option>";
+    }
+
+    if(!$mysqli->query($sqlQuery)){
+        die("An error occured ".$mysqli->error);
```

Comment:
You should return a status 500 with the error message here so that the frontend will know that there is a server error.